packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -723,24 +729,54 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 			return [];
 		}
 
-		// The routes or blob node paths are in the same format as returned in getGCData -
-		// `/<BlobManager.basePath>/<blobId>`.
-		for (const route of sweepReadyBlobRoutes) {
-			const pathParts = route.split("/");
-			assert(
-				pathParts.length === 3 && pathParts[1] === BlobManager.basePath,
-				0x586 /* Invalid blob node id in deleted routes. */,
-			);
-			const blobId = pathParts[2];
+		this.deleteBlobs(sweepReadyBlobRoutes);
+		return Array.from(sweepReadyBlobRoutes);
+	}
+
+	/**
+	 * Delete blobs with the given routes. The routes are GC nodes paths of format `/<BlobManager.basePath>/<blobId>`.
+	 * The blob ids are all local ids. Deleting the blobs involves 2 steps:
```

Comment:
> The blob ids are all local ids

As discussed, when loading a snapshot made by the current code, some could be storage IDs.

I think you should bump the version (removing the feature gate) in this PR and then we should make that other fix before the next release.

Reply:
If we haven't rolled out the current GC Version bump all the way by the time this release happens, then I guess we can revert the GC version in the release branch or something for Loop's sake.  But hopefully we will.  Maybe this is too cowboy, dunno.